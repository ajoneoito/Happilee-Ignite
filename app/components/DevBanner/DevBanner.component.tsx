import {View, Text} from 'react-native';
import React from 'react';
import {DevBannerStyle} from './DevBanner.style';
const DevBanner = () => {
  return (
    <View style={DevBannerStyle.container}>
      <Text>Dev Preview</Text>
    </View>
  );
};

export default DevBanner;
