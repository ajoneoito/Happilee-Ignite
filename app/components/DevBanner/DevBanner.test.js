import React from 'react';
import renderer from 'react-test-renderer';
import DevBanner from './DevBanner.component';

test('renders correctly', () => {
  const tree = renderer.create(<DevBanner />).toJSON();
  expect(tree).toMatchSnapshot();
});
