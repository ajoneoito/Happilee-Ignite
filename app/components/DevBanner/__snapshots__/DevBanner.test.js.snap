// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F9627D",
      "borderRadius": 6,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginHorizontal": 20,
      "padding": 10,
    }
  }
>
  <Text>
    Dev Preview
  </Text>
</View>
`;
